GameData/ROCapsules/PartConfigs/MercuryBDB/MercuryCM.cfg

```diff
@@ -523,11 +532,19 @@ PART
 	}
 }
 
+//	================================================================================
+//	Override RO global settings to get our temperatures back
+//	================================================================================
+@PART[ROC-MercuryCMBDB]:FOR[RealismOverhaul]
+{
+	@skinMaxTemp = 1144
+}
+
 //	================================================================================
 //	Final Pass to Make Sure TAC does not add extra resources
 //	================================================================================
 
-@PART[ROC-MercuryCM]:BEFORE[zzzRealismOverhaul]
+@PART[ROC-MercuryCMBDB]:BEFORE[zzzRealismOverhaul]
```

Comment:
Oof.  Reminder to the rest of us that we need to be careful about when we make new parts versus just renaming something.  Yay not breaking saves.  Boo for creating patches that no longer get applied.

Reply:
This looks like a comment rather than a work item, so I'm resolving.